meta-lmp-base/recipes-bsp/u-boot/u-boot-ostree-scr-fit/boot-common.cmd.in

```diff
@@ -172,7 +167,7 @@ if test "${fiovb.is_secondary_boot}" = "0"; then
 
 	# That means that we've validated already boot image over secondary path
 	# So we can update stable primary path with that images
-	if test "${fiovb.bootupgrade_available}" = "1" && test "${fiovb.upgrade_available}" = "0"; then
+	if test ${fiovb.bootupgrade_available} -gt 0 && test "${fiovb.upgrade_available}" = "0"; then
```

Comment:
This test will fail with an OTA where the boot was not finalized because at that point `upgrade_available = 1`.

I think we need to add a "finalization" check for `bootupgrade_available > 1` somewhere up above that includes a reset similar to how the alternate path handles it.

Reply:
What exactly we should do during finalization?